docs/test-lanes.md

```diff
@@ -0,0 +1,6 @@
+#TEST LANES LABELS
```

Comment:
There is a README file under the `tests` folder, I think it is more appropriate to add this information there.

But I am not sure the information here is relevant for running the tests using `make functest` locally.
There are two "API" for running the e2e tests, one from the CI and one locally. If you want to provide documentation about it, there is a need to clarify which one you use and how one should use it all the way through (i.e. what to call and with what parameters).

Reply:
@EdDev I have updated the `tests/README.md` and deleted the 'docs/test-lanes.md`.